.travis.yml

```diff
@@ -79,11 +79,10 @@ install:
   # Import the fixture, if it exists.
   - phing remember -Dversion=$VERSION -Ddb.database=drupal -Ddb.user=lightning -Ddb.password=lightning -Ddb.host=127.0.0.1
   - cd docroot
-  - drush cache-rebuild
```

Comment:
I think it was for safety. I needed to remove it because in some cases, without the updates having been run, cache clearing will fail.